code/game/turfs/simulated/floor/plasteel_floor.dm

```diff
@@ -8,6 +8,15 @@
 	. = ..()
 	. += "<span class='notice'>There's a <b>small crack</b> on the edge of it.</span>"
 
+// SKYRAT EDIT - SANITARY
+/turf/open/floor/plasteel/Exit(atom/movable/mover, atom/newloc)
```

Comment:
Okay yeah I looked at the code and it's absolutely not good to be using ``Exit`` for this.

Exit is called on all turfs that ATTEMPT to enter the tile which means that if I am on a tile, and I move against a wall, it can trigger dirt. What you actually want is ``Exited`` but even then that's not a good idea.

I recommend finding out if the human subtype has some sort of post_move proc or something that is called when a person successfully moves across a turf. Right now the type check is calling on every single movable atom that moves, including things that aren't human types. Also I highly recommend refactoring how cleanable dirt works because this tries to create a new object every time even though it shouldn't. Like I said it should directly modify the dirt instead of trying to create a new instance of dirt.